docs/core/tools/dotnet-build.md

```diff
@@ -61,7 +67,57 @@ The project file to build. If a project file is not specified, MSBuild searches
 
 ## Options
 
-# [.NET Core 2.x](#tab/netcore2x)
+# [.NET Core 2.1](#tab/netcore21)
+
+`-c|--configuration {Debug|Release}`
+
+Defines the build configuration. The default value is `Debug`.
+
+`-f|--framework <FRAMEWORK>`
+
+Compiles for a specific [framework](../../standard/frameworks.md). The framework must be defined in the [project file](csproj.md).
+
+`--force`
+
+Forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is the same as deleting the *project.assets.json* file.
+
+`-h|--help`
+
+Prints out a short help for the command.
+
+`--no-dependencies`
+
+Ignores project-to-project (P2P) references and only builds the specified root project.
+
+`--no-incremental`
+
+Marks the build as unsafe for incremental build. This flag turns off incremental compilation and forces a clean rebuild of the project's dependency graph.
+
+`--no-restore`
+
+Doesn't execute an implicit restore during build.
+
+`-nodeReuse:false`
```

Comment:
@livarcocc @wlii3

I'm a bit confused about why this is showing up in docs, but not in help and why it isn't Posix style.  Any thoughts?

Reply:
Tagging @wli3

Reply:
It's not POSIX-style because it's an MSBuild passthrough argument, and we went with single-dash/single-slash commands (on Windows it was `/` only).

That might mean that this isn't a great place to put it. There's already a mention that

> In addition to its options, the `dotnet build` command accepts MSBuild options, such as `/p` for setting properties or `/l` to define a logger. For more information about these options, see the [MSBuild Command-Line Reference](/visualstudio/msbuild/msbuild-command-line-reference).

Which may be sufficient? If CLI users need to know about the node-reuse switch explicitly, it's probably a good idea t link to the MSBuild docs from that subsection as well.

Reply:
Thanks @rainersigwald. I think that should be enough in my opinion. Otherwise, we'd have to add the entire MSBuild reference in this topic. We could add information on the remarks if this particular option is extremely important for and common for dotnet-build.